docs/.ruby-version

```diff
@@ -1 +1 @@
-2.7.0
+2.7.1
```

Comment:
This is coming in another PR, but I added it so my build doesn't fail on Netlify.